apps/examples/stdk_smart_switch/Makefile

```diff
@@ -0,0 +1,183 @@
+DEVICE_HEADER_BACKUP_FILE=$(DEVICE_HEADER_FILE).bak
+PRODUCT_JSON_FILE=$(ST_APP_PATH)/onboarding_config.json
+PRODUCT_HEADER_FILE=$(ST_APP_PATH)/onboarding_config.h
+PRODUCT_HEADER_BACKUP_FILE=$(PRODUCT_HEADER_FILE).bak
+
+CFLAGS+=-I$(STDK_SRC_DIR)/include/
+CFLAGS+=-I$(STDK_SRC_DIR)/include/bsp
+CFLAGS+=-I$(STDK_SRC_DIR)/include/os
+CFLAGS+=-I$(STDK_SRC_DIR)/include/mqtt
+CFLAGS+=-I$(STDK_SRC_DIR)/port/os/tizenrt
+
+ASRCS =
+CSRCS =
+MAINSRC = smart_switch.c device_control.c
+
+AOBJS = $(ASRCS:.S=$(OBJEXT))
+COBJS = $(CSRCS:.c=$(OBJEXT))
+MAINOBJ = $(MAINSRC:.c=$(OBJEXT))
+
+SRCS = $(ASRCS) $(CSRCS) $(MAINSRC)
+OBJS = $(AOBJS) $(COBJS)
+
+ifneq ($(CONFIG_BUILD_KERNEL),y)
+  OBJS += $(MAINOBJ)
+endif
+
+ifeq ($(CONFIG_WINDOWS_NATIVE),y)
+  BIN = ..\..\libapps$(LIBEXT)
+else
+ifeq ($(WINTOOL),y)
+  BIN = ..\\..\\libapps$(LIBEXT)
+else
+  BIN = ../../libapps$(LIBEXT)
+endif
+endif
+
+ifeq ($(WINTOOL),y)
+  INSTALL_DIR = "${shell cygpath -w $(BIN_DIR)}"
+else
+  INSTALL_DIR = $(BIN_DIR)
+endif
+
+CONFIG_EXAMPLES_PROGNAME ?= stdk_smart_switch$(EXEEXT)
+PROGNAME = $(CONFIG_EXAMPLES_PROGNAME)
+
+ROOTDEPPATH = --dep-path .
+
+# Common build
+
+VPATH =
+
+all: .built
+.PHONY: clean depend distclean
+
+$(AOBJS): %$(OBJEXT): %.S
+	$(call ASSEMBLE, $<, $@)
+
+$(COBJS) $(MAINOBJ): %$(OBJEXT): %.c
+	$(call COMPILE, $<, $@)
+
+.built: $(OBJS)
+	$(call ARCHIVE, $(BIN), $(OBJS))
+	@touch .built
+
+ifeq ($(CONFIG_BUILD_KERNEL),y)
+$(BIN_DIR)$(DELIM)$(PROGNAME): $(OBJS) $(MAINOBJ)
+	@echo "LD: $(PROGNAME)"
+	$(Q) $(LD) $(LDELFFLAGS) $(LDLIBPATH) -o $(INSTALL_DIR)$(DELIM)$(PROGNAME) $(ARCHCRT0OBJ) $(MAINOBJ) $(LDLIBS)
+	$(Q) $(NM) -u  $(INSTALL_DIR)$(DELIM)$(PROGNAME)
+
+install: $(BIN_DIR)$(DELIM)$(PROGNAME)
+
+else
+install:
+
+endif
+
+ifeq ($(CONFIG_BUILTIN_APPS)$(CONFIG_EXAMPLES_STDK_SMART_SWITCH),yy)
+$(BUILTIN_REGISTRY)$(DELIM)$(FUNCNAME).bdat: $(DEPCONFIG) Makefile
+	$(Q) $(call REGISTER,$(APPNAME),$(FUNCNAME),$(THREADEXEC),$(PRIORITY),$(STACKSIZE))
+
+context: $(BUILTIN_REGISTRY)$(DELIM)$(FUNCNAME).bdat
+####################################
+# Convert device & product profile
+#####################################
+	$(Q) cp -n $(STDK_SRC_DIR)/include/profile/json/device_info_example.json $(DEVICE_JSON_FILE)
+	$(Q) xxd -i $(DEVICE_JSON_FILE) > $(DEVICE_HEADER_FILE); echo $$?;
+	$(Q) sed -i.bak 's/_.*json/device_info/g' $(DEVICE_HEADER_FILE)
+	$(Q) rm $(DEVICE_HEADER_BACKUP_FILE)
+	$(Q) cp -n $(STDK_SRC_DIR)/include/profile/json/onboarding_config_example.json $(PRODUCT_JSON_FILE)
+	$(Q) xxd -i $(PRODUCT_JSON_FILE) > $(PRODUCT_HEADER_FILE); echo $$?;
+	$(Q) sed -i.bak 's/_.*json/onboarding_config/g' $(PRODUCT_HEADER_FILE)
+	$(Q) rm $(PRODUCT_HEADER_BACKUP_FILE)
+
+else
+context:
+
+endif
+
+.depend: Makefile $(SRCS)
+	@$(MKDEP) $(ROOTDEPPATH) "$(CC)" -- $(CFLAGS) -- $(SRCS) >Make.dep
+	@touch $@
+
+depend: .depend
+
+clean:
+	$(call DELFILE, .built)
+	$(call CLEAN)
+	$(Q) rm $(DEVICE_HEADER_FILE)
+	$(Q) rm $(PRODUCT_HEADER_FILE)
```

Comment:
Let's use the same command.
```
$(call DELFILE, $(PRODUCT_HEADER_FILE))
```

Reply:
fixed, and also for smart_lamp example, thanks.